mne/simulation/raw.py

```diff
@@ -0,0 +1,614 @@
+                    for d in dig if d['kind'] == FIFF.FIFFV_POINT_HPI])
+        chpi_rrs = [d['r'] for d in dig if d['kind'] == FIFF.FIFFV_POINT_HPI]
+        R, r0 = fit_sphere_to_headshape(raw.info, verbose=False)[:2]
+        R /= 1000.
+        r0 /= 1000.
+
+        blink_rr = [d['r'] for d in raw.info['dig']
+                if d['ident'] == FIFF.FIFFV_POINT_NASION][0]
+        blink_rr = blink_rr - r0
+        blink_rr = (blink_rr / np.sqrt(np.sum(blink_rr * blink_rr)) *
+                0.98 * R)[np.newaxis, :]
+        blink_rr += r0
+        blink_bem = make_sphere_model(
+            r0, head_radius=R, relative_radii=(0.99, 1.), sigmas=(0.33, 0.33),
+            verbose=False)
+
+    if blink:
+        blink_rate = np.cos(2 * np.pi * 1. / 60. * raw.times)
+        blink_rate *= 12.5 / 60.
+        blink_rate += 4.5 / 60.
+        blink_times = rng.rand(raw.n_times) < blink_rate / raw.info['sfreq']
+        # vary amplitudes
+        blink_times = blink_times * (rng.rand(raw.n_times) + 0.5)
+        # Convolve blink times with kernel to get blink traces
+        blink_kernel = np.hanning(int(0.25 * raw.info['sfreq']))
+        blink_data = np.convolve(blink_times, blink_kernel,
+                                 'same')[np.newaxis, :]
+        blink_data += rng.randn(blink_data.shape[1]) * 0.05
+        blink_data *= 100e-6  # Scale to blink EMG scale
+        del blink_times,
+
+    if ecg:
+        ecg_rr = np.array([[-R, 0, -3 * R]])
+
+        max_beats = int(np.ceil(raw.times[-1] * 70. / 60.))
+        cardiac_idx = np.cumsum(rng.uniform(60. / 70., 60. / 50., max_beats) *
+                                raw.info['sfreq']).astype(int)
+        cardiac_idx = cardiac_idx[cardiac_idx < raw.n_times]
+        cardiac_data = np.zeros(raw.n_times)
+        cardiac_data[cardiac_idx] = 1
+        cardiac_kernel = np.concatenate([
+            2 * np.hanning(int(0.04 * raw.info['sfreq'])),
+            -0.3 * np.hanning(int(0.05 * raw.info['sfreq'])),
+            0.2 * np.hanning(int(0.26 * raw.info['sfreq']))], axis=-1)
+        ecg_data = np.convolve(cardiac_data, cardiac_kernel,
+                               'same')[np.newaxis, :]
+        ecg_data += rng.randn(ecg_data.shape[1]) * 0.05
+        ecg_data *= 3e-4
+        del cardiac_data
+
+    # TODO: This is just scaling to correct physiological scale, correct?
+    #       Can we just roll the scaling into above funcs if statements
+    #       and handle the data assignment there?
+    #       We should also check that the correct channels exist
+
+    # Add to data file, then rescale for simulation
+    '''
+    for data, scale, exg_ch in zip([blink_data, ecg_data],
+                                   [1e-3, 5e-4],
+                                   ['EOG062', 'ECG063']):
+        ch = pick_channels(raw.ch_names, [exg_ch])
+        if len(ch) == 1:
+            raw._data[ch[0], :] = data
+        data *= scale
+    '''
```

Comment:
(add the artifacts if e.g., `blink=True`, but don't add an EOG channel or anything)